UEBPrep/app/views/homepage/index.html.erb

```diff
@@ -1,24 +1,17 @@
-<div id="home-header">
-  <img id="home-logo" src="assets/homepage/logo2.png" alt="image">
+<div class="row">
+  <div id="home-content" class="small-12 small-centered columns">
+	<div class="row"><img src="assets/homepage/iHeader.png" alt="image" data-interchange=""></div>
+	<h1>Unified English Braille eLearning Platform</h1>
+	<div id="home-mailList" class="small-6 small-centered columns">
+		<p><h4>Join our mailing list for project updates!</h4></p><br>
+		<div class="small-8 small-centered columns">
+			<%= form_tag("homepage/subscribe_email", method: "post") do %>
+				<%= email_field_tag(:email_address, nil, :placeholder=>"Email") %>
```

Comment:
I realize now that we don't have the links to sign_up login on this template because we moved it into a partial. yay :+1: